curriculum/teamwork-and-toolkit/schedule.md

```diff
@@ -2,99 +2,108 @@
+[user-manuals-30]: https://github.com/fac20/user-manuals
+[db-colour-30]: http://facresources.com/slides/design-burst-week1.html
+[db-colour-ws-60]: https://github.com/bobbysebolao/learn-css-variables
+[git-ws-120]: https://github.com/foundersandcoders/git-workflow-workshop-for-two
 
 ## Day 3
 
-| Time    | Activity                                 | Learning outcomes |
-| ------- | -----------------------------------------| ----------------- |
-| 09:45   | Check-in                                 |                   |
-| 10:00   | [Learn HTML forms][learn-forms]          | HTML forms        |
-| 11:00   | [Intro to project management][pm-slides] |                   |
-| 11:15   | [Project][projects]                      |                   |
-| _13:00_ | _Lunch_                                  |                   |
-| 14:00   | Project                                  |                   |
-| 17:00   | Speaker                                  |                   |
-
-[learn-forms]: https://github.com/oliverjam/learn-html-forms/
-[projects]: https://founders-and-coders.gitbook.io/coursebook/documents/projects
-[pm-slides]: https://hackmd.io/@fac/S1wGfV2M8#/
+| Time    | Activity                                     | Learning outcomes          |
+| ------- | -------------------------------------------- | -------------------------- |
+| 09:45   | Hello from Charlie, pastoral mentor          |                            |
+| 09:50   | Check-in                                     |                            |
+| 10:00   | [Learn HTML forms][learn-forms-60]           | HTML forms                 |
+| 11:00   | [Intro to project management][pm-slides-15]  | Managing tech projects     |
+| 11:15   | [Begin projects][projects]                   |                            |
+| _13:00_ | _Lunch_                                      |                            |
+| 14:00   | [Projects][projects]                         |                            |
+| 17:45   | Check-out                                    |                            |
 
+[learn-forms-60]: https://github.com/oliverjam/learn-html-forms/
+[pm-slides-15]: https://hackmd.io/@fac/S1wGfV2M8#/
+[projects]: ./project.md
 
 ## Day 4
 
-| Time    | Activity |
-| ------- | -------- |
-| 09:45   | Check-in |
-| 10:00   | Project  |
-| _13:00_ | _Lunch_  |
-| 14:00   | Project  |
+| Time    | Activity                         | Learning outcomes |
+| ------- | -------------------------------- | ----------------- |
+| 09:45   | Hello from Joko, pastoral mentor |                   |
+| 09:45   | Check-in                         |                   |
+| 10:00   | [Projects][project-info]         |                   |
+| _13:00_ | _Lunch_                          |                   |
+| 14:00   | [Projects][project-info]         |                   |
+| 17:45   | Check-out                        |                   |
+[project-info]: ./project.md
```

Comment:
This needs a line break before it otherwise it's part of the table. I'm also not sure relative links to Markdown files like that work on Gitbook. Not sure we ever figured out a solution to this